src/js/port_handler.js

```diff
@@ -152,7 +143,14 @@ PortHandler.check_usb_devices = function (callback) {
                     text: usbText,
                     data: {isDFU: true},
                 }));
+
+                self.portPickerElement.append($("<option/>", {
+                    value: 'manual',
+                    i18n: 'portsSelectManual',
+                    data: {isManual: true},
+                }));
                 self.portPickerElement.val('DFU').change();
+                i18n.localizePage();
```

Comment:
I'm not too sure if localize all the page with any change is the way. I suppose that is for the "manual" port to be translated. Maybe is better to add directly the text in the correct language with `i18n.getMessage('portsSelectManual')`.

@haslinghuis is this `i18n.localizePage()` needed for something more?

Reply:
Had to use it otherwise it was blank, replaced with suggestion.